Leanplum-SDK/Classes/LeanplumRequest.m

```diff
@@ -417,7 +445,7 @@ - (void)sendRequests:(BOOL)async
             if (_error != nil) {
                 _error(err);
             }
-            [sendNowQueue cancelAllOperations];
+            [[LeanplumRequest sendNowQueue] cancelAllOperations];
```

Comment:
technically there should only be always one operation in the queue right?
because only after the response is received, you call [self sendRequests:async] in line 382 again?

is this just for additional protection?